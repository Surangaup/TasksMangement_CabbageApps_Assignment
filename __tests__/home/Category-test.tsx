import * as React from 'react';
import renderer from 'react-test-renderer';
import Category from '../../src/screens/home/category';

test(('Category snapshot'), ()=>{
    const snap = renderer.create(<Category />).toJSON();
    expect(snap).toMatchSnapshot();
})