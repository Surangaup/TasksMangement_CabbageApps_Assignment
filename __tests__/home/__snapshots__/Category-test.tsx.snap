// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Category snapshot 1`] = `
<RCTScrollView
  contentContainerStyle={
    Object {
      "height": 70,
    }
  }
  data={
    Array [
      Object {
        "id": 1,
        "isSelected": true,
        "onPress": [Function],
        "title": "My Day",
      },
      Object {
        "id": 2,
        "isSelected": false,
        "onPress": [Function],
        "title": "Important",
      },
      Object {
        "id": 3,
        "isSelected": false,
        "onPress": [Function],
        "title": "Personal ",
      },
      Object {
        "id": 4,
        "isSelected": false,
        "onPress": [Function],
        "title": "Category 04",
      },
      Object {
        "id": 5,
        "isSelected": false,
        "onPress": [Function],
        "title": "Category 05",
      },
      Object {
        "id": 6,
        "isSelected": false,
        "onPress": [Function],
        "title": "Category 06",
      },
      Object {
        "id": 7,
        "isSelected": false,
        "onPress": [Function],
        "title": "Category 07",
      },
    ]
  }
  disableVirtualization={false}
  extraData={[Function]}
  getItem={[Function]}
  getItemCount={[Function]}
  horizontal={true}
  initialNumToRender={10}
  keyExtractor={[Function]}
  maxToRenderPerBatch={10}
  onContentSizeChange={[Function]}
  onEndReachedThreshold={2}
  onLayout={[Function]}
  onMomentumScrollEnd={[Function]}
  onScroll={[Function]}
  onScrollBeginDrag={[Function]}
  onScrollEndDrag={[Function]}
  removeClippedSubviews={false}
  renderItem={[Function]}
  scrollEventThrottle={50}
  stickyHeaderIndices={Array []}
  updateCellsBatchingPeriod={50}
  viewabilityConfigCallbackPairs={Array []}
  windowSize={21}
>
  <View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#5A3EA4",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              Object {
                "color": "white",
              },
            ]
          }
        >
          My Day
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Important
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Personal 
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Category 04
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Category 05
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Category 06
        </Text>
      </View>
    </View>
    <View
      onLayout={[Function]}
      style={
        Array [
          Object {
            "flexDirection": "row",
          },
          null,
        ]
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F7F8FA",
            "borderRadius": 10,
            "margin": 5,
            "minWidth": 100,
            "opacity": 1,
            "padding": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "black",
              },
              false,
            ]
          }
        >
          Category 07
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
